Documentation update regarding cache utilisation step (#1394)
kyma-project/btp-manager: docs/contributor/04-10-workflows.md

      Pods.
    - Compares the fetched memory usage for the SAP BTP service operator against a predefined threshold.
- **The test fails in the following conditions**:
    - The threshold is exceeded.

</details>

<details>
<summary>Large Labeled Secrets Check</summary>

- **Purpose**: Check the resource consumption (CPU and memory) of BTP Manager and the SAP BTP service operator after the
  creation of large number of labeled Secrets. Labeled secrets are cached by the SAP BTP service operator when the
  LimitedCache feature is enabled, which can lead to an increase in memory usage. This test simulates this scenario to
  ensure that the memory usage remains within acceptable limits.
- **Steps**:
    - Creates additional large labeled Secrets (64KB each) to simulate the memory footprint increase when the operator
      watches labeled resources.
    - Invokes kubectl top command to fetch the CPU and memory usage of BTP Manager and the SAP BTP service operator
      Pods.
    - Compares the fetched memory usage for the SAP BTP service operator against the value from previous step.

> - **Memory footprint is expected to increase** after the creation of large labeled Secrets, as the SAP BTP service

Reviewer: ```suggestion
- **Expected result**:
   - Memory footprint increases after creating large labeled Secrets, as the SAP BTP service
```
